utils/db_utils.py

```diff
@@ -94,3 +94,18 @@ def query_confirmed_trips(start_date, end_date):
     # logging.debug("After filtering, the actual data is %s" % df.head())
     # logging.debug("After filtering, the actual data is %s" % df.head().trip_start_time_str)
     return df
+
+
+def add_user_stats(user_data, trip_data):
+    trip_df = pd.DataFrame(trip_data)
+    trip_grouped = trip_df.groupby('user_id')
+    for user in user_data:
+        user_id = user['user_id']
+        if user_id in trip_grouped.groups:
+            trips = trip_grouped.get_group(user_id)
+            user['confirmed_trips'] = len(trips)
```

Comment:
This seems to be `total_trips`.
`confirmed_trips` aka `labeled_trips` should be the number of trips that have labels.
I have existing script that essentially generates that this table needs
https://github.com/e-mission/e-mission-server/blob/master/bin/monitor/check_participant_status.py

Unfortunately, it uses `edb` directly, so please import `edb` in this function only (so you are not tempted to use it elsewhere) and file a new issue to add timeseries support for the functionality needed here.